packages/components/navbar/src/Navbar.tsx

```diff
@@ -78,15 +78,19 @@ function _Navbar(props: ExpandProps<NavbarProps>, ref: React.Ref<HTMLElement>) {
           )}
         </Flex>
         <Flex alignItems="center" gap="spacingXs">
-          {switcher}
-          {secondaryNavigation && (
-            <Flex aria-label="Secondary Navigation">{secondaryNavigation}</Flex>
-          )}
-          {account && (
-            <Flex aria-label="Account Navigation" gap="spacingXs">
-              {account}
-            </Flex>
-          )}
+          <Flex alignItems="center">{switcher}</Flex>
```

Comment:
ah this sneaked up from the other PR, is to make it easier if we need to show/hide elements based on screen size, since we can add classNames this way

Reply:
I had it commited, forgot to push 😞